src/main/resources/application.conf

```diff
@@ -43,6 +43,9 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+  local {
+    tryToHardlinkInputs = true
+  }
```

Comment:
Well the SGE backend now exists.  Yay!  How does that effect the comments above and/or the mergeability of this PR?